sample-config/compose/plane.yml

```diff
@@ -0,0 +1,84 @@
+version: "3.9"
+services:
+  drone:
+    container_name: plane-drone
+    image: ghcr.io/drifting-in-space/plane-drone:latest
```

Comment:
the `image` declared here seems to be ignored in favor of building, we probably want to use the image by default.

The ideal behavior is to use the image unless `docker compose build` is explicitly called, but I'm not sure if `docker compose` allows that.